invisu.csl

```diff
@@ -574,11 +673,13 @@
           <text term="ibid" text-case="capitalize-first" font-style="italic"/>
         </else-if>
         <else-if position="subsequent">
-          <text macro="author"/>
+          <text macro="author-sub"/>
           <group delimiter=", ">
             <text macro="disambiguate"/>
-            <group delimiter="&#160;">
-              <text macro="locators"/>
+            <group delimiter=" ">
+              <text term="note" prefix="(" />
+              <text variable="first-reference-note-number" suffix=")"/>
+              <text macro="locators"/>       
             </group>
```

Comment:
It would be much better to rewrite this as:

```xml
            <group delimiter=" ">
              <group prefix="(" suffix=")" delimiter=" ">
                <text term="note"/>
                <text variable="first-reference-note-number"/>
              </group>
              <text macro="locators"/>       
            </group>
```